custom_components/tuya_local/devices/zigbee_zn373186_temphumid_sensor.yaml

```diff
@@ -0,0 +1,37 @@
+name: Climate sensor
+products:
+  - id: xr3htd96
+    name: Temperature Humidity Sensor (zn373186_01)
```

Comment:
Brand name Tenkey? Based on #1146

Reply:
looks like is other name Aubess, but since it is the same, you can keep the name from this MR.

Reply:
Tenkey is the name of the seller selling these in aliexpress. But there's no brand in the package or in the product. I believe it's Aubess https://iothelper.tistory.com/entry/%EC%95%8C%EB%A6%AC%EC%9D%B5%EC%8A%A4%ED%94%84%EB%A0%88%EC%8A%A4-IoT-Zigbee-%EC%98%A8%EC%8A%B5%EB%8F%84%EA%B3%84-tuya-ZN37318601-%EC%8A%A4%EB%A7%88%ED%8A%B8%EC%8B%B1%EC%8A%A4-%EC%97%B0%EA%B2%B0-1